04-rstudio-tour.Rmd

```diff
@@ -1,38 +1,277 @@
+
+For Mac users, note that you will need to move the RStudio icon into the icon that looks like the Applications folder. 
+
+
+Drag and drop RStudio into the Applications folder to install on a Mac
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Drag and drop RStudio into the Applications folder to install on a Mac."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_58")
+```
+
+See[here](https://jhudatascience.org/intro_to_r/modules/Setup/R_and_RStudio_installation.html) for more information on the process of installing RStudio.
+
+If you run into trouble, check the following:
+
+- did you install the correct version of software for your operating system?
+   - Check that you installed the version right for your type of system, (`macOS` vs `Windows` for example)
+   - Check if maybe you need a different version for the age of your system. First check that your version of R was right - there are multiple versions for different `macOS` systems for example. You can check the apple icon (top left corner) and "About This Mac" to learn more about the age of your operating system. 
+   
+If your operating system is older (and you can't update it), try installing progressively older versions found [here](https://www.rstudio.com/products/rstudio/older-versions/) until it works. You will know if it worked if you try to open RStudio and you see an interface without a message about things going poorly. Here you can see an [example](https://community.rstudio.com/t/rstudio-desktop-crashes-on-startup-with-library-not-loaded/130296) of this. 
+
+</details>
+
+### Updating RStudio
+
+It is also a good idea to keep RStudio up-to-date. New features become available as the Posit team works on developing RStudio. So if you already have RStudio, you might want to check to see if your version is up-to-date.
+
+To check for updates you can go to the `Help` menu at the top of RStudio and then click on `Check for Updates`. 
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Check for updates for RStudio in the Help menu of RStudio."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_50")
+```
+
+## Navigating RStudio
+
+Now that you hopefully have RStudio running on your machine, we will walk you through some of the major features that can really help you with your data analyses.
+
+### Default Layout
+
+First it is important to be familiar with the layout. When you first open RStudio you will see 3 panes.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "When RStudio is first opened you will see 3 panes"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_67")
+```
+
+<details><summary>If your RStudio looks different click here.</summary>
```

Comment:
```suggestion
<details><summary>If you'd like to change your RStudio layout click here</summary>
```
If it looks different, its because they've already changed their layout and are familiar with RStudio. So perhaps this section should be moved after you've explained the panes and instead be more of an FYI for how to change the layout?

Reply:
I have had students accidentally change it and not realize it. Like they started with R years ago and changed their defaults and forgot :P